idaes/generic_models/properties/core/generic/generic_property.py

```diff
@@ -833,11 +833,15 @@ def define_metadata(cls, obj):
         # created by state var methods
         obj.add_properties(
             {'flow_mol': {'method': None},
-             'flow_mol_phase': {'method': None},
-             'flow_mol_phase_comp': {'method': None},
-             'flow_mol_comp': {'method': None},
-             'mole_frac_comp': {'method': None},
-             'mole_frac_phase_comp': {'method': None},
+             'flow_vol': {'method': '_flow_vol'},
+             'flow_mass': {'method': '_flow_mass'},
+             'flow_mass_phase': {'method': '_flow_mass_phase'},
+             'flow_vol_phase': {'method': '_flow_vol_phase'},
+             'flow_mol_phase': {'method': '_flow_mol_phase'},
+             'flow_mol_phase_comp': {'method': '_flow_mol_phase_comp'},
+             'flow_mol_comp': {'method': '_flow_mol_comp'},
+             'mole_frac_comp': {'method': '_mole_frac_comp'},
+             'mole_frac_phase_comp': {'method': '_mole_frac_phase_comp'},
```

Comment:
I also wonder if we should add scaling factors for these as well, but that could be a separate PR.

Reply:
Yeah.  We should do that.  There are so many flows the logic for calculating the missing scale factors will be interesting.